### PR TITLE
Add a fixer to remove constructor return tag

### LINKED ISSUE #12: cs-fixer: remove constructors return tag.
I propose to add a cs-fixer which remove the PHPDoc `@return` tag of constructors.
Since they are [optional](http://www.phpdoc.org/docs/latest/references/phpdoc/tags/return.html) and current `@return void` is invalid, it can be a good thing :)

### PR DESCRIPTION
**Pull request by Metalaka**
Fix #12

### CODE REVIEW

File: Resource/PHPCSFixer/Fixer/PhpdocConstructorReturn.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+class PhpdocConstructorReturn extends AbstractFixer
+{
+    const CONSTRUCTOR_NAME = '__construct';
+
+    public function fix(SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
+
```
Comment (Hywan) [inline Resource/PHPCSFixer/Fixer/PhpdocConstructorReturn.php]: CS: No empty line here.

File: Resource/PHPCSFixer/Fixer/PhpdocConstructorReturn.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            $docBlock = new DocBlock($token->getContent());
```
Comment (Hywan) [inline Resource/PHPCSFixer/Fixer/PhpdocConstructorReturn.php]: CS: Align `=` please (everywhere).

**Comment by Hywan** [comment]
Why is it that complicated? Can't use something like (in pseudo-code):

```
if ($method is CONSTRUCT_NAME) {
    if ($docBlock = $method->getDocBlock()) {
        $docBlock->removeTag('return');
    }
}
```

I don't understand why you reach the `T_FUNCTION` token, manage several line blocks etc.

**Comment by Metalaka** [comment]
i refactored the code to be more readable, but I can't make it as simple as you write.

**Comment by Hywan** [comment]
Looks correct! Thanks :-).

**Comment by Hywan** [comment]
Tested on https://github.com/hoaproject/Acl/pull/18, it works great!

**Pull request merged**